addons/mail/static/src/core/store_service.js

```diff
@@ -53,6 +53,16 @@ export class Store {
      */
     guest = null;
 
+    /**
+     * The last id of bus notification at the time for fetch init_messaging.
+     * When receiving a notification:
+     * - if id greater than this value: the notification is newer than init_messaging state.
+     * - if same id or lower: the notification is older than init_messaging state.
+     * This is useful to determine whether we should increment or decrement a counter based
+     * on init_messaging state.
+     */
+    initMessagingLastNotifId = 0;
```

Comment:
suggestion shorter name: `initBusId`?

Reply:
And it would be nice if the key in the payload of init messaging was the same name as the field.